src/test/java/core/basesyntax/StoreStorageTest.java

```diff
@@ -21,7 +21,7 @@
     public final static String FIRST_OUTPUT_DATA = "banana,20,2020-10-17";
 
     @Test
-    public void storageTestSupplyWithEmptyStorage() {
+    public void storageTest_supply_emptyStorage() {
```

Comment:
```suggestion
    public void supply_mptyStorage() {
```
you don't need to write class name before method, same with other method in test class